apps/lab/src/app/global.css

```diff
@@ -1,3 +1 @@
-@import url('../../node_modules/@uireact/foundation/dist/index.css'); 
 
```

Comment:
What is the theme being rendered in the lab app?

Reply:
Looks like there is no theme there:
<img width="1728" alt="Screenshot 2024-12-17 at 7 13 58 PM" src="https://github.com/user-attachments/assets/e808e423-9972-492f-8813-bf958a23fb8b" />

Reply:
My bad, i'm gonna add again the styles